packages/front-b2b-link-rn/README.md

```diff
@@ -1,3 +1,82 @@
 # Front Finance React Native SDK
 
-TODO
\ No newline at end of file
+JS library for integrating with Front Finance.
+
+### Install
+
+With `npm`:
+
+```
+npm install --save @front-finance/frontfinance-rn-sdk
+```
+
+With `yarn`
+
+```
+yarn add @front-finance/frontfinance-rn-sdk
+```
+
+### Getting connection link
+
+Connection link should be obtained from the GET `/api/v1/cataloglink` endpoint. Api reference for this request is available [here](https://integration-api.getfront.com/apireference#tag/Integrations/paths/~1api~1v1~1cataloglink/get). Request must be preformed from the server side because it requires the client secret. You will get the response in the following format:
```

Comment:
for transfer UX it's POST catalog link, I'd keep an API reference only, btw the link looks broken

Reply:
The content has been taken from [front-b2b-link](https://github.com/FrontFin/front-b2b-link/blob/main/packages/link/README.md). I have updated the link reference. Feel free to leave a suggestion.